capstone/backend/src/main/java/com/google/spanner/LibraryFunctions.java

```diff
@@ -35,14 +36,20 @@
   private static String DATABASE_INSTANCE = "capstone-instance";
   private static String DATABASE_NAME = "step39-db";
   
-  private static final String EMAIL =     "Email";
-  private static final String ITEMTYPES = "ItemTypes";
-  private static final String LISTID =    "ListId";
-  private static final String LISTNAME =  "DisplayName";
-  private static final String USERID =    "UserId";
-  private static final String USERLISTS = "UserLists";
-  private static final String USERNAME =  "Username";
-  private static final String USERS =     "Users";
+  private static final String ADDRESS =          "Address";
+  private static final String EMAIL =            "Email";
+  private static final String ITEMID =           "ItemId";
+  private static final String ITEMNAMEANDBRAND = "ItemNameAndBrand";
+  private static final String ITEMTYPES =        "ItemTypes";
+  private static final String LISTID =           "ListId";
+  private static final String LISTNAME =         "DisplayName";
```

Comment:
String DISPLAY_NAME = "DisplayName"? Name and variable name should be consistent.